### PR TITLE
Put caikit's base config back into the container

### PR DESCRIPTION
**Pull request by Xaenalt**
> Put caikit's base config back into the container
> Rename caikit-*.yaml to caikit-tgis*.yaml to better reflect actual state
> Update a few docs as well

### CODE REVIEW

**Comment by Xaenalt** [comment]
@dtrifiro PTAL

**Comment by dtrifiro** [comment]
Can you rebase this? It's likely only related to the apiversion changes

File: demo/kserve/custom-manifests/caikit/caikit-tgis-servingruntime.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      #   requests:
+      #     cpu: 8
+      #     memory: 16Gi
+    - name: transformer-container
+      image: quay.io/opendatahub/caikit-tgis-serving:stable
+      env:
+        - name: RUNTIME_LOCAL_MODELS_DIR
+          value: /mnt/models
+      ports:
+        - containerPort: 8080
```
Comment (bdattoma) [inline demo/kserve/custom-manifests/caikit/caikit-tgis-servingruntime.yaml:29]: @Xaenalt it doesn't work for me if I use port  8080, it returns me HTTP invalid request error in the pod logs. It works if I use port 8085. Could you pls double check?

Comment (dtrifiro) [reply demo/kserve/custom-manifests/caikit/caikit-tgis-servingruntime.yaml:29]: @bdattoma: try changing:

```diff
diff --git a/demo/kserve/custom-manifests/caikit/caikit-tgis-servingruntime.yaml b/demo/kserve/custom-manifests/caikit/caikit-tgis-servingruntime.yaml
index 8528909..9fee801 100644
--- a/demo/kserve/custom-manifests/caikit/caikit-tgis-servingruntime.yaml
+++ b/demo/kserve/custom-manifests/caikit/caikit-tgis-servingruntime.yaml
@@ -21,7 +21,7 @@ spec:
       #     cpu: 8
       #     memory: 16Gi
     - name: transformer-container
-      image: quay.io/opendatahub/caikit-tgis-serving:stable
+      image: quay.io/opendatahub/caikit-tgis-serving:fast
       env:
         - name: RUNTIME_LOCAL_MODELS_DIR
           value: /mnt/models
```

Comment (dtrifiro) [reply demo/kserve/custom-manifests/caikit/caikit-tgis-servingruntime.yaml:29]: See here:

https://github.com/caikit/caikit-nlp/pull/244/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R77-R82

**Pull request merged**